plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -371,7 +415,7 @@ def run_inspection(file_filter, options, do_extract, file_store_tmp, file_store_
           file_filter.matches?("/" + dir)
         end
       end
-      managed, unmanaged = dirs.keys.partition{ |d| rpm_dirs.has_key?(find_dir + d) }
+      managed, unmanaged = dirs.keys.partition{ |d| inspected_dirs.has_key?(find_dir + d) }
```

Comment:
`Hash#has_key?` is deprecated in favor of `Hash#key?`.<br>Space missing to the left of {.